### ISSUE TITLE
Pivot Grid feature does not work with hidden fields

### ISSUE THREAD

**Issue by data-goblin** [TabularEditor/TabularEditor3#345]
Hi there, 

Noticed today a possible issue with the Pivot Grid feature & hidden fields (TE Enterprise 3.1.7). 
Apologies if this is the intended behavior: 

_**If a table or column is hidden, the values will not load / show in the pivot grid.**_
This can be observed when adding a field to the pivot grid (i.e. 'Date'[Year]), hiding it & saving changes, then refreshing the pivot grid. Unhiding the field causes the values to re-appear. This is observed with both columns & measures. For measures, hiding the "home table" containing the measure does not have an impact, however. 

-----------------------------------

**See attached images:**

Unhidden table; Unhidden Column ✔
![TE_Pivot-Grid_Unhidden-Table_Unhidden-Column](https://user-images.githubusercontent.com/97248341/149092312-d71d150b-2ba8-4b33-a4c8-7c6227bac04f.png)

Unhidden table; Hidden column ❌
![TE_Pivot-Grid_Unhidden-Table_Hidden-Column](https://user-images.githubusercontent.com/97248341/149092310-2ec16215-f35d-4592-9a4d-3e2f4dc9528f.png)

Hidden table; Unhidden column ❌
![TE_Pivot-Grid_Hidden-Table_Unhidden-Column](https://user-images.githubusercontent.com/97248341/149092315-cb8aca95-1c31-4e87-aa37-a9b3664372fd.png)

-----------------------------------

Unhidden table; Unhidden measure ✔
![TE_Pivot-Grid_Unhidden-Table_Unhidden-Measure](https://user-images.githubusercontent.com/97248341/149092314-c87fffb1-87a3-4839-940e-88c75e1b842e.png)

Unhidden Table; Hidden Measure ❌
![TE_Pivot-Grid_Unhidden-Table_Hidden-Measure](https://user-images.githubusercontent.com/97248341/149092311-6749f3a7-c6d8-4f66-bd3a-e7f27b005a55.png)

Hidden table; Unhidden Measure ✔
![TE_Pivot-Grid_Hidden-Table_Unhidden-Measure](https://user-images.githubusercontent.com/97248341/149092317-5b4189ad-6df1-48b0-a8e5-4047d5b5f4e0.png)

**Comment by otykier** [comment]
We'll look into it - thanks for reporting.

**Comment by edhans** [comment]
Yup. Came here to report the same thing. Unhiding the field fixes it. This is in 3.2.2
![HiddenFieldsInPivotGrid](https://user-images.githubusercontent.com/45298358/158275196-86a8e228-7b5d-4aef-80f4-835b3fd1bf6e.gif)

**Comment by mlonsk** [comment]
The same problem exists for measures, but instead of not being added they are just shown as having Error:
![image](https://user-images.githubusercontent.com/11939114/161998134-6d277b73-cced-4861-9642-0aef3db86cf1.png)

If the measure is turned to visible the Pivot grid works again
